packages/expo/src/MailComposer/ExponentMailComposer.web.ts

```diff
@@ -0,0 +1,33 @@
+import array from 'cast-array';
+import filter from 'object-filter';
+import qs from 'query-string';
+
+export default {
+  get name() {
+    return 'ExponentMailComposer';
+  },
+  async composeAsync(options) {
+    const check = value => (value ? array(value).join(',') : undefined);
```

Comment:
Pull this out into module scope since we don't need to create a new function per compose call. Also unless you specifically need `undefined`, return `null` to indicate the lack of a value.